components/more_channels/more_channels.tsx

```diff
@@ -40,7 +40,7 @@ export type Props = {
     teamId: string;
     teamName: string;
     channelsRequestStarted?: boolean;
-    bodyOnly?: boolean;
+    bodyOnly?: boolean; // TODO remove me
```

Comment:
Not strictly related to this PR, but I removed the equivalent prop from MoreDirectChannels as part of this
https://mattermost.atlassian.net/browse/MM-38449